2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,30 @@
+let list = document.getElementById("todo-list");
+
+while (list.hasChildNodes()) {
+  list.removeChild(list.firstChild);
+}
+
 function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
+  // let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  todos.forEach((todo) => {
```

Comment:
looping through your todos like this a great start, to make your code more readable & resusable you could create a function called something like `createToDo(todo)` which could be called with your for loop